src/functions.lisp

```diff
@@ -183,6 +196,8 @@
 ;;; %foreign-funcall-varargs on some hypothetical lisp on an
 ;;; hypothetical platform that has different calling conventions for
 ;;; varargs functions. :-)
+;;;
+;;; -- indeed it is useful for ECL static FFI!
```

Comment:
Since it's no longer hypothetical, I've simply removed the comment in f696f3db4ac75e6a516e3a238a28cf1c327776f2.